### PR TITLE
Move and rename upload button on entity sidepanel

### LINKED ISSUE #3557: Move the "Upload new file" button inline and to the right of the main "Documents" header
- [ ] Rename "Documents" to "Primary Documents".
- [ ] Rename "Upload new file" to "Upload PDF".
- [ ] Move green upload button inline and to the right of this header (so that it is consistent with the "Add supporting files" button in the area below it).

The info icon and tooltip can be added at a future time.

### PR DESCRIPTION
**Pull request by grafitto**
fixes #3557 

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

**Comment by grafitto** [comment]
image
Here is how it looks like

**Comment by grafitto** [comment]
image

File: app/react/Attachments/components/AttachmentsList.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class AttachmentsList extends Component {
     const attachments = AttachmentsList.arrangeFiles(this.props.attachments);
     return (
       <div className="attachments-list-parent">
-        <div className="attachments-list-header">
+        <div className="attachments-list-header" style={{ alignItems: 'center' }}>
```
Comment (konzz) [inline app/react/Attachments/components/AttachmentsList.js:34]: lets avoid inline styles.

File: app/react/Attachments/components/scss/filelist.scss [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.filelist-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  div {
+    label {
+      width: 162px;
+      margin-top: 5px !important
```
Comment (konzz) [inline app/react/Attachments/components/scss/filelist.scss:8]: avoid using !important

Comment (grafitto) [reply app/react/Attachments/components/scss/filelist.scss]: Removed

File: app/react/Attachments/components/AttachmentsList.js (hunk 1 above)
Comment (grafitto) [reply app/react/Attachments/components/AttachmentsList.js]: Removed

**Pull request merged**